lib/slack/renderer/push.js

```diff
@@ -13,24 +13,32 @@ class Push extends Message {
     this.push = push;
   }
 
+  get commits() {
+    return this.push.commits.map((commit) => {
+      const message = commit.message.split('\n')[0];
+      const sha = commit.id.substring(0, 8);
+      return `<${commit.url}|\`${sha}\`> - ${message}`;
+    }).join('\n');
+  }
+
   getCore() {
     const noOfCommits = this.push.commits.length;
     const commits = noOfCommits === 1 ? 'commit' : 'commits';
     const branch = `\`${this.push.ref.replace('refs/heads/', '')}\``;
     const pushed = this.push.forced ? 'force-pushed' : 'pushed';
     const text = `<${this.push.compare}|${noOfCommits} new ${commits}> ${pushed} to ${branch}`;
     return {
-      fallback: text,
       text,
     };
   }
 
   getRenderedMessage() {
     return {
+      ...this.getCore(),
```

Comment:
This "pretext" way of showing the number of commits is very different to how we've been making use of the pretext field so far. It's typically ${github object} ${past tense action} by ${actor}. For example `Pull request merged by bkeepers`.
So instead of showing this message above the attachment, could we move it inside the attachment, maybe as a title or **bolded** first line before list of commits?


If you really want it to stay this way, this line should probably be placed inside the attachment as 
```
...this.getCore()
```

with the return value of this.getCore() being:
```
{
    fallback: text,
    pretext: text,
}
```